x/src/main.rs

```diff
@@ -46,6 +47,11 @@ enum Command {
 }
 
 fn main() -> Result<()> {
+    let env = Env::default()
+        .filter_or("MY_LOG_LEVEL", "info")
+        .write_style_or("MY_LOG_STYLE", "always");
+    env_logger::init_from_env(env);
+
```

Comment:
```suggestion
    env_logger::from_env(env_logger::Env::default().default_filter_or("info")).init();
```

If you really, really want you can add a `-q` flag to `x` and just not enable the logger if that flag is set

Reply:
I discovered this locally as well, as I wanted to default to local, my current approach has a special formatter to make the logging not as visually jarring, and I _think_ should honor RUST_LOG as well (at least for the level).   Maybe there's a way to pull in cargo's default logging style, but didn't see it.

Reply:
actually you were further ahead,  cloning the local env seemed to be the only way to honor configs, or override with info.   I guess I better get used to this rust with immutable objects.